browser/src/index.tsx

```diff
@@ -294,7 +294,7 @@ const start = (args: string[]) => {
     UI.init()
 
     ipcRenderer.on("menu-item-click", (_evt, message) => {
-        instance.command("normal! " + message)
+        instance.command("exec \":normal! " + message + "\"")
```

Comment:
@justinmk, I tried implementing this with `feedkeys` but `\<c-r>` still doesn't get interpreted.  Any idea why I can't send a Ctrl key to `feedkeys()`?